packages/ODEs/src/app.ts

```diff
@@ -576,6 +578,8 @@ export class DiffStudio {
 
   /** Solve IVP */
   private async solve(ivp: IVP, inputsPath: string): Promise<void> {
+    const noCustomSettings = (ivp.solverSettings === DEFAULT_SOLVER_SETTINGS);
```

Comment:
Better to avoid negation in naming, as this leads to confusion; I would have called it `customSettings`

Reply:
Fixed